### DISCUSSION TITLE
Channel Search

### DISCUSSION THREAD

**Discussion by matthias811** [Yogui79/IntexPureSpa#20] [General]
First, thanks for providing the manual and code - I hope I will get this working!
Nevertheless I am stuck on Step 1, the channel search. The light is blinking continously and console is showing me all the channel numbers but I cannot detect a flickering like shown in your video. What is the distance the LC12s should be away from the pool? I tried a few meters away. Must there be something activated on the pool or is it just ok to have it on power with pump running?
I have ordered two LC12s by different accounts (to ensure one is there in time) and tried now with both, same effect....

**Comment by LEECHER1** [comment]
Hey Matthias,

LC12s ist blinking when search. Flickering when to get a signal and found a ID.

The distance from the Pumpe not more as 5m but better for the ID Search max. 1m. 
When the SPA is normal running is ok. You can tray a pumpe restart and tray again to search.

Please check again all the connections LC12s and ESP32.

Maybe you can help this: (last commented)
https://github.com/Yogui79/IntexPureSpa/issues/17

**Reply by djgel** [reply]
see topic #10  , I have a  same  problem, !

**Reply by matthias811** [reply]
I reflashed with 3s delay and tried two cycles. I have placed the LC12s only 2cm above the remote control on the pump. I have the model SC-20, picture djgel postet in #10 looks equal to mine.
No success until now... 
The link for ESP32 you provided did not work for me (out of stock), I bought this one but should be OK, right? (https://www.amazon.de/gp/product/B074RGW2VQ/ref=ppx_yo_dt_b_asin_title_o04_s00?ie=UTF8&psc=1)

**Reply by djgel** [reply]
@matthias811 , 
I recorded a video of how the light blinked very dimly! Remember to leave the control on the pump and on!! I swear it was very difficult to find, I only found it putting 5 seconds and the environment VERY dark! Follow link! 

https://youtu.be/4lLdoyjJGHQ

It must be the same problem as me!

about ESP32 (chip is not the 32D), your kit appears v1, for me the code Spa.ino didn't work, giving panic!

**Reply by Yogui79** [reply]
the blink looks good coold you please provide the arduino ide monitor when blinking

**Reply by djgel** [reply]
@Yogui79  video is mine ! I found a channel and network, now  have another problem ... I sad this to  @matthias811 for help! :)

**Reply by Yogui79** [reply]
oups;)

**Reply by matthias811** [reply]
Thanks a lot for providing the hints and the video Márcio. I will plan a night session the next days, my pool is outside in the sun. The normal blinking I could see pretty good but if the flickering is with less intensity this could maybe be the issue. In addition I have ordered another ESP32 with v4. Maybe this as well makes an difference.

**Comment by matthias811** [comment]
It works now! Thanks for helping me.
I used now the ESP32 v4 and searched with 5seconds delay. Luckily mine is on channel 10, did not take long to find finally. The networks search was about 2h to find the network ID.

**Comment by darti2** [comment]
I also thank you very much for the description. I'm just getting started, but I'm a total beginner.

It might be a very stupid question, but I'm asking it because it's not clear to me.

When searching for a channel, there is flashing in the first step, on the serial monitor it is at Chanel 42.
The question is, is this already a hex code, or will I still have to convert it to a hex code?

**Reply by homerisback** [reply]
Maybe a late reply (2y later) but you put it 0x42 and your done.
